tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -1176,18 +1105,25 @@ def test_restart_unsafely(self):
 
     @cluster(num_nodes=16, log_allow_list=RESTART_LOG_ALLOW_LIST)
     def test_rolling_restarts(self):
-        self._lifecycle_test_impl(self._rolling_restarts)
+        self._lifecycle_test_impl(
+            # It takes ~4mins to safely restart a node, hence, we limit the total
```

Comment:
I mean this is just really bad

Reply:
The time to restart or the limit? The time to restart includes:
- enabling maintenance mode
- waiting for leadership to drain
- waiting for no unavailable partitions
- restarting the RP process
- disabling maintenance mode
- waiting for leadership to be balanced
- waiting for no under-replicated partitions

It did seem high to me as well, however, some of the above can take a minute or so.